apps/accounts/management/commands/fix_permissions.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: ai ts=4 sts=4 et sw=4
+
+"""
+Project: bluebutton-web-server
+App: apps.accounts.management.commands.
+FILE: fix_permissions
+Created: 3/14/18 3:59 PM
+
+Created by: '@ekivemark'
+"""
+import sys
+
+from django.contrib.auth.management import _get_all_permissions
+from django.contrib.auth.models import Permission
+from django.contrib.contenttypes.models import ContentType
+from django.core.management.base import BaseCommand
+from django.apps import apps
+
+
+class Command(BaseCommand):
+    help = "Fix permissions for proxy models."
```

Comment:
It looks like this command rebuilds all permissions from settings and all models. If that's right the help text should be updated.

Reply:
fixed help description and made another push.